.travis.yml

```diff
@@ -17,7 +17,19 @@ bundler_args: --jobs 7 --retry 3
 matrix:
   include:
     - rvm: 2.5.1
-    - env: "GEMFILE_MOD=\"gem 'chef', '= 13.8.3'\""
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 14.3.37'\""
+      rvm: 2.5.1
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 14.2.0'\""
+      rvm: 2.5.1
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 14.1.1'\""
+      rvm: 2.5.1
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 14.0.202'\""
+      rvm: 2.5.1
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 13.10.0'\""
+      rvm: 2.5.1
+    - env: "GEMFILE_MOD=\"gem 'chef', '= 13.9.4'\""
+      rvm: 2.5.1
```

Comment:
Did we rev the ruby version in these?

Reply:
I'm assuming we should have whatever is the stable version of Ruby and the stable release of chef, iirc all of them are `2.5.1` so I _should_ be able to remove them all?